pages/ppr-marketing.vue

```diff
@@ -47,55 +51,61 @@
 
     <v-container class="body-text">
       <h1 class="text--center" style="padding-top: 80px">Coming Soon...</h1>
+
       <v-row class="pt-3 pb-8">
         <v-col cols="12" class="text--center">
           In early 2022, the BC Personal Property Registry will be launching an
           all-new application.
         </v-col>
       </v-row>
+
       <v-row style="padding-bottom: 80px">
         <v-col class="text--center px-6 col-md-3 col-sm-6">
           <img class="pr-1" src="@/assets/svgs/table-search.svg" />
-          <v-icon medium color="#38598a">mdi-currency-usd-circle</v-icon
```

Comment:
I think this v-icon was a typo so I deleted it. It didn't display because that's not a valid MDI icon.

Anyway, this block looks correct with the img element above.